### ISSUE TITLE
Lines extend beyond viewable area for .txt files

### ISSUE THREAD

**Issue by 0scubasteve0** [nextcloud/text#1006]
**Describe the bug**
Each line in a text file extends infinitely to the right until broken manually by the enter/return key in files with a .txt extension.

**To Reproduce**
Open existing text file with a .txt extension or start new file and enter an unbroken line of text. 

**Expected behavior**
Lines wrap based on the size of the editor window as they do with .md files. 

**Screenshots**

Web
![text-web](https://user-images.githubusercontent.com/36998292/92270704-815b7500-eeb4-11ea-9b94-438f49754105.gif)
![text-web2](https://user-images.githubusercontent.com/36998292/92270717-83bdcf00-eeb4-11ea-89b9-a08ab145c109.gif)

Android App
![text-mobile](https://user-images.githubusercontent.com/36998292/92270953-f5961880-eeb4-11ea-9cf4-97d9bba60f0c.gif)

**Server Info**
Nextcloud 19.02

**Client Info**
Firefox 78-80
macOS, Linux
Nextcloud Android App v3.13.0

**Comment by dave-kennedy** [comment]
I'd love to take a stab at fixing this - perhaps adding an option to wrap long lines. I just need to get my head around how the code is organized and whatnot first.

**Comment by juliushaertl** [comment]
Yes as discussed with @jancborchardt we should generally wrap lines even in plain text files.

**Comment by NielBuys** [comment]
This problem affect me to.

**Comment by juliushaertl** [comment]
@NielBuys  Please use the [Github Reactions](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments) feature instead of commenting that also need a specific feature. That way we can see your support without notifying all users subscribed to a given issue.

**Comment by alabre** [comment]
Hi there, is there any news on this topic? Running NC 21 still shows the behaviour with the scroll bar, except I missed something in the settings. Without line wrapping the text editor is somewhere between hardly usable and unusable and unfortunately shared .txt files are opened with it by default.

There was a line wrapping before in NC 18, so is it a degression or a feature request to have no line wrapping?
Will line wrapping come back? If so, are there new plans when?

All the best and thank you

**Comment by pwnoobz** [comment]
I am sorry if this is fixed. I am on the latest Next Cloud stable version - 22.2.3 and I can confirm having the same problem unless there is an option for word wrap that I'm not seeing. But I'd love to have this behaviour as the default for text files.

**Comment by dave-kennedy** [comment]
@pwnoobz this was released with Nextcloud 23. See #1883 and linked milestone.

**Comment by azul** [comment]
This was meant to be backported as it's a bugfix but the backport failed. I'll give it another try.
